int16 max_num_repeate
---
bool success
